CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Flutter Plugin Changelog
 
+## Version 4.0.0 - September 21, 2020
+Major release for Airship Android SDK 14.0.0 and iOS SDK 14.1.1.
```

Comment:
Look at the react-native release for the changelog i added. We need to warn people about the behavior change with the url allow list.